src/pyrokinetics/gk_code/gs2.py

```diff
@@ -72,6 +83,15 @@ class GKInputGS2(GKInput, FileReader, file_type="GS2", reads=GKInput):
         "beta_prime": 0.0,
     }
 
+    pyro_gs2_mxh_defaults = {
+        **pyro_gs2_miller_defaults,
+        "cn": [0.0, 0.0, 0.0, 0.0, 0.0],
+        "sn": [0.0, 0.0, 0.0, 0.0, 0.0],
+        "dcndr": [0.0, 0.0, 0.0, 0.0, 0.0],
+        "dsndr": [0.0, 0.0, 0.0, 0.0, 0.0],
+        "n_mxh": 5,
```

Comment:
I don't know if we actually want to have the number of modes in this data set or if we should automatically determine it from the size of the cn/sn etc. lists.